### PR TITLE
feat: update to version v0.4.20 of ActSVG

### PR DESCRIPTION
**Pull request by asalzburger**
There have been a few fixes and updates to the `actsvg` library, mainly to allow batched surface displays, e.g. all frontside vs all backside module.

This PR increases the version and adapts the code.

### CODE REVIEW

File: Plugins/ActSVG/include/Acts/Plugins/ActSVG/TrackingGeometrySvgConverter.hpp [hunk 1]
```diff
@@ -66,6 +66,31 @@ void convert(const GeometryContext& gctx, const TrackingVolume& tVolume,
...
+  std::vector<ActsScalar> rzEtaLines;
+};
+
+/// Convert into xy and zr projections only
+///
+/// @param gctx the geometry context
+/// @param tVolume the tracking volume
+/// @param cOptions the conversion options
+///
+/// @note best performant if stuffen wirh the right options
```
Comment (AJPfleger) [inline Plugins/ActSVG/include/Acts/Plugins/ActSVG/TrackingGeometrySvgConverter.hpp:86]:
```suggestion
/// @note best performant if stuffed with the right options
```

File: Plugins/ActSVG/src/EventDataSvgConverter.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                                                           unsigned int idx) {
+  return point<actsvg::views::x_y>(pos, size, style, idx);
+}
+
+actsvg::svg::object Acts::Svg::EventDataConverter::pointZR(const Vector3& pos,
+                                                           ActsScalar size,
+                                                           const Style& style,
+                                                           unsigned int idx) {
+  return point<actsvg::views::z_r>(pos, size, style, idx);
+}
```
Comment (AJPfleger) [inline Plugins/ActSVG/src/EventDataSvgConverter.cpp:23]:
```suggestion
}
```

File: Examples/Io/Svg/include/ActsExamples/Io/Svg/SvgDefaults.hpp [hunk 3]
```diff
@@ -46,12 +79,43 @@ trackingGeometryOptions() {
...
+
+  return tgOptions;
+};
+
 }  // namespace
 
 namespace ActsExamples {
 
-static Acts::Svg::Style s_defaultLayerStyle = layerStyle();
+static Acts::Svg::Style s_layerStyle = layerStyle();
```
[the remark this answers is not in the document]
Comment (asalzburger) [reply Examples/Io/Svg/include/ActsExamples/Io/Svg/SvgDefaults.hpp:109]: Removed, indeed.

File: Examples/Io/Svg/include/ActsExamples/Io/Svg/SvgPointWriter.hpp [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+/// Write out any 3D point collection in Svg format.
+///
+/// @tparam TColl is the collection type
+/// @tparam Acc is the accessor type for x(), y(), z()
+///
+///
+/// This writes one file per event into the configured output directory. By
+/// default it writes to the current working directory. Files are named
+/// using the following schema
```
[the remark this answers is not in the document]
Comment (asalzburger) [reply Examples/Io/Svg/include/ActsExamples/Io/Svg/SvgPointWriter.hpp:70]: Fixed.

File: Examples/Io/Svg/include/ActsExamples/Io/Svg/SvgPointWriter.hpp [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+///
+///     event000000001-spacepoints.svg
+///     event000000002-spacepoints.svg
+///
+template <typename T, typename Acc = AccessorXYZ>
+class SvgPointWriter final : public WriterT<GeometryIdMultiset<T>> {
+ public:
+  struct Config {
+    std::string writerName =
+        "PointWriter";  ///< the name of the writer (for basee initialization)
```
[the remark this answers is not in the document]
Comment (asalzburger) [reply Examples/Io/Svg/include/ActsExamples/Io/Svg/SvgPointWriter.hpp:80]: Removed, did not make sense.

File: Plugins/ActSVG/include/Acts/Plugins/ActSVG/TrackingGeometrySvgConverter.hpp (hunk 1 above)
Comment (asalzburger) [reply Plugins/ActSVG/include/Acts/Plugins/ActSVG/TrackingGeometrySvgConverter.hpp:86]: Changed comment

File: Plugins/ActSVG/src/EventDataSvgConverter.cpp (hunk 2 above)
Comment (asalzburger) [reply Plugins/ActSVG/src/EventDataSvgConverter.cpp:23]: newline added.

**Review by AJPfleger — APPROVED** [review]

**Pull request merged**